### PR TITLE
fix(operating-license-application): Submit operatingLicenseApplication and update syslumenn service

### PR DESCRIPTION
**Pull request by alexdiljar**
Changing uploadData in syslumenn service to receive multiple attachments. Had to update a some applications to do so. 

Adding submit to syslumenn in operating license template.

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by johannaagma — APPROVED** [review]

File: libs/clients/syslumenn/src/lib/syslumenn.spec.ts [hunk 1]
```diff
@@ -43,10 +43,12 @@ const PERSON = [
     type: PersonType.Plaintiff,
   },
 ]
-const ATTACHMENT = {
-  name: 'attachment',
-  content: 'content',
-}
+const ATTACHMENT = [
```
Comment (RunarVestmann) [inline libs/clients/syslumenn/src/lib/syslumenn.spec.ts:46]: Maybe this could be renamed to ATTACHMENTS to indicate that it's a list

**Review by RunarVestmann — APPROVED** [review]
Looks good to me, I left a minor comment which is up to you if you'd like to address it or not (it's just a simple suggestion regarding a variable name change). I'll approve :)

**Review by eirikurn — APPROVED** [review]
LGTM

**Review by obmagnusson — APPROVED** [review]
LGTM. But you might want to update the codeowners file for these template-api's

**Review by kksteini — APPROVED** [review]

**Pull request merged**